test/e2e/node/pod_hostnamefqdn.go

```diff
@@ -63,15 +63,15 @@ func testPod() *v1.Pod {
 	return pod
 }
 
-var _ = SIGDescribe("Hostname of Pod [Feature:SetHostnameAsFQDN]", func() {
+var _ = SIGDescribe("Hostname of Pod [Feature:SetHostnameAsFQDN][NodeAlphaFeature:SetHostnameAsFQDN]", func() {
```

Comment:
Do you need the `[Feature:SetHostnameAsFQDN]` tag? I can't find any jobs in kubernetes/test-infra that specifically mention it

Reply:
it was an original change:

![image](https://user-images.githubusercontent.com/9950081/91239827-45c2ed00-e6f5-11ea-85d2-5d353b3b922f.png)


Based on @javidiaz reply, adding Feature removing it from one of the queries that runs as part of a CI.

@javidiaz did you figure out which test is running these ginko tests? I'm curious if adding Feature is the right fix here.


I think this is lgtm even with Feature enabled. At least we will start running these tests as part of node-kubelet-alpha test.

/lgtm

Reply:
@spiffxp Without having Feature tag, some CIs that do not have Alpha features enabled run them (e.g. pull-kubernetes-e2e-kind). I found other NodeAlphaFeature e2e tests that have both Feature and NodeAlphaFeature (e.g., TTLAfterFinished, ShareProcessNamespace, CPUManager). I am happy to change things to whatever you think it is right.

@SergeyKanzhelev I do not think any of the tests checking this PR are running these ginko tests right now.

Reply:
🤦  right, thanks, that makes sense in context.  To dedupe, we'd have to go update a bunch of skip regexes to now also include `\[NodeAlphaFeature:.+\]` and hey why not `\[NodeSpecialFeature:.+\]` while we're at it.

Instead of having to change a bunch of jobs, would it make more sense to instead use `[NodeAlphaFeature][Feature:Foo]` and update the few node jobs that test these feature?

Reply:
(I'm fine with this change being done as a followup since it requires a few cross-repo PR's to get it right, just want to get consensus and a followup issue open before putting this into the merge pool)

Reply:
I really like `[NodeAlphaFeature][Feature:Foo]` approach going forward.

Reply:
i can help with the follow up PR

Reply:
Thanks guys. Just FYI there are also tests with [NodeFeature:.+][Feature:.+] tags. 

Now that 1.19 is out I think we can merge this, right? :)

Reply:
opened https://github.com/kubernetes/kubernetes/issues/94289